.github/workflows/integration_test.yml

```diff
@@ -16,37 +16,35 @@ jobs:
       - uses: actions/setup-go@main
         with:
           go-version: 1.18
+          
       - name: install kubectl
         run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+
       - name: start local k8s cluster
         run: make cluster-up
         env:
             CLUSTER_PROVIDER: ${{matrix.kube_provider}}
-      - name: Checkout kube-operator repo
-        uses: actions/checkout@v3
-        with:
-          repository: prometheus-operator/kube-prometheus
-          path: kube-prometheus
-      - name: deploy kube operator
-        run:  ls && kubectl create -f manifests/setup && until kubectl get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done && kubectl create -f manifests/
-        working-directory: kube-prometheus
+            PROMETHEUS_ENABLE: "true"
+            GRAFANA_ENABLE: "false"
+
       - name: simple test - deploy kepler
         run: make cluster-sync
         env:
             CLUSTER_PROVIDER: ${{matrix.kube_provider}}
             CTR_CMD: docker
-      - name: Login to Quay
```

Comment:
do you have another GH action to push the image?

Reply:
I will uncomment this, but should it be the image-base.yml?
Push a new image shouldn’t be an action of integration test....

Reply:
sounds another PR :D